taichi/codegen/codegen_opengl.h

```diff
@@ -30,7 +32,8 @@ class OpenglCodeGen {
 
   Program *prog_;
   Kernel *kernel_;
-  StructCompiledResult *struct_compiled_;
+  [[maybe_unused]] StructCompiledResult *struct_compiled_;
```

Comment:
BTW, do you observed unused warnings about Metal members on non-Mac platforms? If so, maybe I should add this too.

Reply:
No, I didn't..

Reply:
Ack. I do see this warning, so it should be necessary.